inference-engine/cmake/templates/InferenceEngineDeveloperPackageConfig.cmake.in

```diff
@@ -22,32 +26,46 @@ endforeach()
 message("")
 
 # for samples in 3rd party projects
-set(gflags_DIR "@gflags_BINARY_DIR@")
+set_and_check(gflags_DIR "@gflags_BINARY_DIR@")
 
-# Targets
+#
+# Content
+#
 
-if(USE_SYSTEM_PUGIXML)
-    find_package(PugiXML REQUIRED)
-    set_property(TARGET pugixml PROPERTY IMPORTED_GLOBAL TRUE)
-endif()
+find_dependency(IEDevScripts
+                PATHS "${OpenVINO_MAIN_SOURCE_DIR}/cmake/developer_package"
+                NO_CMAKE_FIND_ROOT_PATH
+                NO_DEFAULT_PATH)
+
+find_dependency(InferenceEngine
+                PATHS "${CMAKE_CURRENT_LIST_DIR}"
+                NO_CMAKE_FIND_ROOT_PATH
+                NO_DEFAULT_PATH)
+
+# WA for cmake: it exports ngraph as IE::ngraph in the IE export list
+# while we already have ngraph export in its own export list as ngraph::ngraph
+set_property(TARGET ngraph::ngraph PROPERTY IMPORTED_GLOBAL TRUE)
```

Comment:
It already looks like `OpenVINODeveloperPackageConfig.cmake` 😄

Reply:
good point, we can rename it in a separate PR after all CF activities